src/text-editor-component.js

```diff
@@ -2118,6 +2125,7 @@ class TextEditorComponent {
       // rendered start row accurately. 😥
       this.populateVisibleRowRange(renderedStartRow)
       this.props.model.setEditorWidthInChars(this.getScrollContainerClientWidthInBaseCharacters())
+      this.derivedDimensionsCache = {}
```

Comment:
Did this fix some other failing seed or something?

Reply:
This is cherry-picked from your PR which was not yet merged when this was opened https://github.com/atom/atom/pull/15828 just to verify that PR wouldn't fix this failure.